src/components/nodes/sequenceFlow/sequenceFlow.vue

```diff
@@ -20,52 +20,17 @@ export default {
       shape: null,
       definition: null,
       sourceShape: null,
+      target: null,
+      validNodeColor: '#dffdd0',
+      invalidNodeColor: '#fae0e6',
+      defaultNodeColor: '#fff',
+      gatewayDirectionOptions: { Diverging: 'Diverging', Converging: 'Converging' },
```

Comment:
Instead of the `sequenceFlow` component owning this data, I think it should be in the gateway config.

`/src/components/nodes/exclusiveGateway/index.js` still has the hard-coded strings for `"Diverging"` and `"Converging"`. You could place the direction options in there, and export it, so the same values can be used in the gateway component and also imported and used in `sequenceFlow.vue`.